ocaml/xapi/certificates.ml

```diff
@@ -396,18 +396,23 @@ let pool_uninstall kind ~__context ~name =
 
 (* Extracts the server certificate from the server certificate pem file.
    It strips the private key as well as the rest of the certificate chain. *)
-let get_server_certificate () =
-  match Gencertlib.Pem.parse_file !Xapi_globs.server_cert_path with
+let get_public_certficate_from_pkcs12 path =
```

Comment:
For me `get` implies more effort than a `read`. In some way this is true because we are parsing the file but I would still this consider a read, like a database or filesystem read. I would use `get` for something that involves a network request or user interaction.